core/test/uix/core_test.cljs

```diff
@@ -55,16 +56,16 @@
 
 (deftest test-lazy
   (async done
```

Comment:
Apparently, my auto-formatting changed the indentation of this unrelated `async` form. Shall I revert it to the previous form? The new indentation feels a bit more idiomatic, though.